### ISSUE TITLE
Battle Royale Gamerule

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#3104]
tl;dr a shrinking circle like the ones from battle royale games would be useful.

Currently in SSS you could just hide in a locker as an innocent and pretty much win. Alternatively sometimes people are AFK the entire round and it's hard for the traitors to find them if they don't know the map well enough. 

One solution is to just have a shrinking circle to force people to get together and it could help make the game more dynamic.

It would need:
* Shader for the circle (you could copy CircleMask or GradientCircleMask and go from there) so anything outside of the radius has the effect.
* System to manage it shrinking (client-side to know the radius and server-side to know who to damage)
* GameRule so any game mode can use it that wants
* Potentially the ability to make it a StationEvent.

**Comment by PJB3005** [comment]
You'd need a way bigger map for this to ever be useful.
